setup.py

```diff
@@ -58,17 +58,13 @@
         "rt",
         "HarvestingKit>=0.3",
         "mixer==4.9.5",
-        "ipython",
-        "Babel>=1.3",
```

Comment:
I suspect we may still need Babel from a clean installation. If you look at the the `travis.yml` it reveals a few more versioned dependencies we need to add here it seems..